src/NUnitFramework/nunit.framework.legacy/StringAssert.cs

```diff
@@ -292,5 +292,101 @@ public static void DoesNotMatch(string pattern, string actual)
             DoesNotMatch(pattern, actual, string.Empty, null);
         }
         #endregion
+
+        #region IsNullOrEmpty
+
+        /// <summary>
+        /// Asserts that a string is either null or empty.
+        /// </summary>
+        /// <param name="actual">The string to be examined</param>
+        /// <param name="message">The message to display in case of failure</param>
+        /// <param name="args">Arguments used in formatting the message</param>
+        public static void IsNullOrEmpty(string? actual, string message, params object?[]? args)
+        {
+            Framework.Assert.That(actual, Is.Null.Or.Empty, () => ConvertMessageWithArgs(message, args));
```

Comment:
There should be no need to use the `Framework.` prefix. Can you please remove it here and below.